benchmarks/operator_benchmark/pt/add_test.py

```diff
@@ -67,5 +67,48 @@ def forward(self):
 op_bench.generate_pt_test(add_long_configs + add_short_configs, AddmmBenchmark)
 op_bench.generate_pt_gradient_test(add_long_configs + add_short_configs, AddmmBenchmark)
 
+addr_configs = op_bench.cross_product_configs(
+    M=[8, 256],
+    N=[256, 16],
+    device=['cpu', 'cuda'],
+    dtype=[torch.double, torch.half],
```

Comment:
torch.float and torch.double go through the same codepath (different from torch.half). Since other benchmarks use only one type, I thought this would be sufficient.